core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -425,7 +428,7 @@ private[spark] class SparkSubmit extends Logging {
             case (localResources, resolvedUri) =>
               val source = new File(localResources.getPath)
               val dest = new File(
-                ".",
+                workingDirectory,
                 if (resolvedUri.getFragment != null) resolvedUri.getFragment else source.getName)
```

Comment:
Since `.` is participated in, can we call `File#getCanonicalFile` before assigning `source` and `dest` here to avoid the following message?

```log
Files  ... from /tmp/spark-1a68de84-0fb2-4cf6-8f7a-302c24509935/xyz-plugin-spark3-2.12-20230516-131757.jar.jar to /opt/spark/work-dir/./xyz-plugin-spark3-2.12-20230516-131757.jar
```

And the logging statement can be simplified to

```patch
-logInfo(
-  s"Files  $resolvedUri " +
-    s"from ${source.getAbsolutePath} to ${dest.getAbsolutePath}")
+logInfo(s"Files $resolvedUri from $source to $dest")
```

Reply:
refactored logging format, thanks

Reply:
addressed, thanks